framework/contracts/native/account-factory/src/commands.rs

```diff
@@ -136,7 +138,7 @@ pub fn execute_create_account(
         })?;
     }
 
-    let salt = generate_account_salt(&account_id);
+    let salt = module::generate_instantiate_salt(&account_id);
```

Comment:
can we move salt gen to its own file/mod?

weird to call module mod here

Reply:
> can we move salt gen to its own file/mod?
> 
> weird to call module mod here

Sure, sorry I can't stop think of account contracts as modules.

Reply:
moved!